src/routes/image.routes.ts

```diff
@@ -44,12 +44,52 @@ const router = express.Router();
  */
 
 router.get(
-  "/:id",
+  "/:recipeId",
   imageValidator.validateReadRecipeImagesData(),
   validate,
   imageController.readRecipeImages
 );
 
+/**
+ * @swagger
+ * /image/:imageId/display:
+ *   get:
+ *     security:
+ *       - Bearer: []
+ *     tags:
+ *       - image
+ *     description: Returns recipe image as png file
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         schema:
+ *           type: string
+ *           example: 60f7ea20cf60ae0004307aa2
+ *         description: Id of recipe
+ *
+ *     responses:
+ *       200:
+ *         description: Found image
+ *         content:
+ *           application/json:
```

Comment:
This is wrong content type, for 200 you're responding with content type of `image/png`

Reply:
change content type according to areks guides